### PR TITLE
add hypothesis-ros in docs section "external strategies"

### PR DESCRIPTION
**Pull request by fkromer**

### CODE REVIEW

File: hypothesis-python/docs/strategies.rst [hunk 1]
```diff
@@ -24,6 +24,7 @@ Some packages provide strategies directly:
 * :pypi:`hs-dbus-signature` - strategy to generate arbitrary
   `D-Bus signatures <https://dbus.freedesktop.org>`_.
 * :pypi:`hypothesis_sqlalchemy` - strategies to generate :pypi:`SQLAlchemy` objects.
+* :pypi:`hypothesis-ros` - strategies to generate messages and paramters for the `Robot Operating System <http://www.ros.org/>`_.
```
Comment (jjgreen) [inline hypothesis-python/docs/strategies.rst]: paramters -> parameters :-)

**Comment by Zac-HD** [comment]
:tada: Great to see this published!  I'll be very happy to merge this once you fix (or tell me to fix) the typo that jjgreen spotted.

**Comment by fkromer** [comment]
@Zac-HD it just needs to bei fixed:  `paramters` -> `parameters` :smile: . I could fix and squash later as well.

**Comment by fkromer** [comment]
hope w.o. squash is ok as well

**Pull request merged**